source/_patterns/02-molecules/blocks/timeline-item/timeline-item.twig

```diff
@@ -1,9 +1,15 @@
-<div class="jcc-timeline-item">
+<li class="jcc-timeline-item">
   <h3 class="jcc-timeline-item__title">{{ title }}</h3>
   <div class="jcc-timeline-item__excerpt">{{ excerpt }}</div>
 
-  {% if read_more_link %}
-  <a href="{{ url }}" class="jcc-timeline-item__trigger"{% if url_attributes %}{{ url_attributes }}{% endif %}>{{ read_more_link }}</a>
-  {% endif %}
+  {% if read_more.url %}
+    <a href="{{ read_more.url }}" class="jcc-timeline-item__trigger"{% if url_attributes %}{{ url_attributes }}{% endif %} title="{{ read_more.label }}" target="_blank">{{ read_more.label }}</a>
+  {% elseif read_more.id %}
+    {% include '@organisms/sections/full-modal/full-modal.twig' with {
+      modal_id: read_more.id,
+      button_label: read_more.label
+    } %}
```

Comment:
Everything needs to be explicitly passed when nesting components, otherwise they won't print outside of Pattern Lab.
```twig
content: read_more.content 
```